library/init/meta/environment.lean

```diff
@@ -167,6 +167,17 @@ meta constant structure_fields : environment → name → option (list name)
 meta constant get_class_attribute_symbols : environment → name → name_set
 /-- The fingerprint of the environment is a hash formed from all of the declarations in the environment. -/
 meta constant fingerprint : environment → nat
+
+/-- Gets the equation lemmas for the declaration `n`. -/
+meta constant get_eqn_lemmas_for (env : environment) (n : name) : list name
```

Comment:
What's the difference between this and https://github.com/leanprover-community/lean/blob/master/library/init/meta/simp_tactic.lean#L121?

Reply:
The difference is that I didn't look for the function there. :smile:

The new functions are better because they operate on the environment directly and don't require a tactic state.  So I've implemented `get_eqn_lemmas_for` using the new function.